### ISSUE TITLE
Add Playbook to Notion

### ISSUE THREAD

**Issue by GouravSardana** [IndianOpenSourceFoundation/dynamic-cli#93]
In this feature, we need these things - 
1. Create a notion account and generate the token. The flow would be like this - 

    - dynamic --notion -> Open up notion login to the browser and ask them to log in.
    
    - After this -> The terminal will show. 'Notion Logged in' or 'Notion logged in failed'

2. Now it's time to upload. Use dynamic --notionupload or nu. Then it will automatically fill up the question to the notion. We have to implement a similar thing like [this](https://github.com/jamalex/notion-py). We have to make this from scratch.

**Comment by NamamiShanker** [comment]
When you mean to implement it from the scratch, you mean we can't use the `notion` library you linked? We have to build the authentication and uploading system on our own?

**Comment by GouravSardana** [comment]
Nops. Authentication - we can used from notion but from the product prospective, we have to reduce the 3rd party package. Because if someday our 1 thing will upgrade then it might effect other packages.

**Comment by NamamiShanker** [comment]
I agree. I was wondering if we can implement browser login for the notion login. A browser window will open upon typing `dynamic -notion`. The user will log in to the notion. The program will receive an authentication token that can then be used to sync playbook data.

**Comment by GouravSardana** [comment]
Yup, that we have to made it from scratch or need to find some other way

**Comment by NamamiShanker** [comment]
Should we store the token locally in some file so that the user doesn't have to log in again until the cookie expires?

**Comment by Souvikns** [comment]
Hey, I have used the notion API before, build [this for fun](https://github.com/Souvikns/Notion-Board). I would like to work on this, would be nice to have some insight.

**Comment by GouravSardana** [comment]
@Souvikns So basically we implemented the login using notion. If you download dynamic and explore --notion then you can able to login. That way you will get auth token. @NamamiShanker implemented that features. 
We have playbook as well --playbook. We need to upload that playbook data to notion. So that in future if we change our computer/Laptop our playbook still remain in sync.

**Comment by Souvikns** [comment]
Hey, so I am using windows and as it does not run on windows I am running on GitHub codespaces, so when I am using the command `dynamic --notion` it shows `login unsuccessful` as there is no chrome. Is there any other way to set it up.

**Comment by GouravSardana** [comment]
Ahh got it. One way is to use VMware and install linux and use the dynamic.

Another simple way is to comment all the dependencies of terminos. That was causing incompatibility. I'm looking for another way but will take some time. You can try second way
